File: Programmers/코딩테스트_입문/GangHyeok/Day04/피자 나눠먹기(2).java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        int[] six = {1, 2, 3, 6};
        
        for(int i=3;i>=0;i--){
            if(n % six[i] == 0){
                answer = six[i];
                break;
            }
        }
        
        answer = n/answer;
        
        return answer;
    }
}

File: Programmers/코딩테스트_입문/GangHyeok/Day04/피자 나눠먹기(2).js
function solution(n) {
  var answer = 0;
  
  const six = [6, 3, 2, 1];
  for(let i of six){
      if(n %i == 0){
          answer = i;
          break;
      }
  }
  
  answer = n/answer
  
  return answer;
}